### ISSUE TITLE
[5.x] Deprecation warnings | Submission Title Format | Email Notifications

### ISSUE THREAD

**Issue by ArnoMoqo** [verbb/formie#1785]
### Describe the bug

Installing a clean Craft 5.0 with Formie 3.0.0-beta.4 i notice these 3 bugs:

**Deprecation warnings:**
After submitting a form, we get those deprecation warnings. I don't think we can fix/change those?
<img width="1857" alt="Scherm­afbeelding 2024-03-27 om 13 54 10" src="https://github.com/verbb/formie/assets/117268595/db9cc556-394e-4b1e-a1aa-899940ff9f80">

**Submission Title Format**
When setting the submission title, the fields don't get displayed as aspected
<img width="1702" alt="Scherm­afbeelding 2024-03-27 om 13 57 00" src="https://github.com/verbb/formie/assets/117268595/cca62a5f-4d15-42e4-a34e-9c03d09b3e46">

**Email Notifications**
When setting the recipient to the email field we get the following error.
```php
Notification email parse error for “To: {field:email} ”. Template error: “Email "{field:email}" does not comply with addr-spec of RFC 2822.” /var/www/html/vendor/symfony/mime/Address.php:54
```

### Steps to reproduce

1. Install a clean Craft CMS v5 with the latest Formie plugin (3.0.0-beta.4)
2. Add a new form and set the email notifications recipient to the email field
3. Set a form label as the submission title format
4. Submit the form in the frond-end

### Form settings

Default formie settings used.

### Craft CMS version

5.0.0

### Plugin version

3.0.0-beta.4

### Multi-site?

No

**Comment by engram-design** [comment]
Good point, we can't actually deprecate that function, so removing that notice.

Other issues should be fixed as well for the next release. To get this early, run `composer require verbb/formie:"dev-craft-5 as 3.0.0-beta.4"`.

**Comment by ArnoMoqo** [comment]
hi @engram-design, this fixed everything from the above. One more issue, in the queue manager we get the following error when sending form notification. But we do get the email, so the email is in fact being send.

The email does not appear in the 'Sent Notifications' tab, not as success or as failed.


```php
Getting unknown property: craft\mail\transportadapters\Smtp::encryptionMethod
```

**Comment by engram-design** [comment]
Looks like Craft have deprecated and removed some properties on the SMTP mailer. Run `composer require verbb/formie:"dev-craft-5 as 3.0.0-beta.4"` again with that fix.

**Comment by ArnoMoqo** [comment]
Works as expected, thanks.

**Comment by engram-design** [comment]
We'll just keep this open in case others find a similar issue!

**Comment by engram-design** [comment]
Fixed in [3.0.0-beta.5](https://github.com/verbb/formie/releases/tag/3.0.0-beta.5)
